src/amber/cli/commands/routes.cr

```diff
@@ -62,6 +64,12 @@ module Amber::CLI
             action: route_match[4]?, pipeline: current_pipe,
             scope: current_scope, uri_pattern: route_match[2]?
           )
+        elsif route_match = route_string.to_s.match(WEBSOCKET_ROUTE_REGEX)
+          build_route(
+            verb: route_match[1]?, controller: route_match[3]?,
+            action: "", pipeline: current_pipe,
+            scope: current_scope, uri_pattern: route_match[2]?
+          )
```

Comment:
It looks like there is only one difference here:
```crystal
build_route(
  verb: route_match[1]?, controller: route_match[3]?,
  action: "", pipeline: current_pipe,
  scope: current_scope, uri_pattern: route_match[2]?
)
```
The `action` is an empty string when it is a websocket. Can this be DRYed up?

Reply:
I was thinking this would be here for future more detailed separation/improvements to HTTP and websocket code paths which wouldn't be shared between both. But sure, can simplify for now, and the () around regexes that I've added at the top make this trivial. Will add a commit soon.

Reply:
Actually, not simple to DRY. If you find a way, please improve upon my patch.